content/en/account_management/org_settings/cross_org_visibility.md

```diff
@@ -1,10 +1,9 @@
 ---
 title: Cross-Organization Visibility
-is_beta: true
 ---
 
 {{< callout url="#" header="false" btn_hidden="true">}}
```

Comment:
If you want the callout box to stand out more, you can remove the `btn_hidden="true"` and the `Join the Beta!` heading shows up. See this page as an example: https://docs.datadoghq.com/agent/fleet_automation/

```suggestion
{{< callout url="#" header="false" btn_hidden="true">}}
```